src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,80 @@
+import cn from 'classnames';
+import { useSearchParams, Link } from 'react-router-dom';
+import { getSearchWith, SearchParams } from '../utils/searchHelper';
+
+const SEXES = {
+  m: 'Male',
+  f: 'Female',
+};
+const CENTURIES = ['16', '17', '18', '19', '20'];
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const query = searchParams.get('query') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+  const sex = searchParams.get('sex') || '';
+
+  const setSearchWith = (params: SearchParams) => {
+    const search = getSearchWith(searchParams, params);
+
+    setSearchParams(search);
+  };
+
+  // Функція для визначення активності посилання
```

Comment:
Remove comment. Try it write obvious code that explains all we need without comments)